RevenueCatUI/Templates/V2/Components/Button/BottomSheetView.swift

```diff
@@ -101,7 +101,12 @@ struct BottomSheetOverlayModifier: ViewModifier {
                         onDismiss: {
                             self.sheetViewModel = nil
                         },
-                        additionalPadding: self.safeAreaInsets
+                        additionalPadding: EdgeInsets(
+                            top: 0,
+                            leading: 0,
+                            bottom: safeAreaInsets.bottom,
+                            trailing: 0
+                        )
```

Comment:
We only really care about the bottom safe area, like we do for the sticky footer:

https://github.com/RevenueCat/purchases-ios/blob/main/RevenueCatUI/Templates/V2/Components/Root/RootView.swift#L51